packages/react-composites/src/composites/CallComposite/LocalDeviceSettings.tsx

```diff
@@ -34,6 +35,32 @@ const getDropDownList = (list: Array<VideoDeviceInfo | AudioDeviceInfo>): IDropd
   return dropdownList;
 };
 
+const getOptionIcon = (type: iconType): JSX.Element | undefined => {
+  const iconStyles = { marginRight: '8px' };
+
+  if (type === 'Camera') {
+    return <CallVideoIcon style={iconStyles} key={'videoIconKey'} />;
+  } else if (type === 'Microphone') {
+    return <MicIcon style={iconStyles} key={'microphoneIconKey'} />;
+  } else if (type === 'Speaker') {
+    return <VolumeUpIcon style={iconStyles} key={'speakerIconKey'} />;
+  } else {
+    return undefined;
+  }
+};
+
+const onRenderTitle = (props: IDropdownOption[] | undefined, iconType: iconType): JSX.Element => {
```

Comment:
`props?: IDropdownOption`

Explicit union with `undefined` is almost never needed.